src/SumoCoders/FrameworkCoreBundle/Service/JsData.php

```diff
@@ -49,6 +48,8 @@ protected function handleRequestStack()
      */
     public function __toString()
     {
+        $this->handleRequestStack();
```

Comment:
constructor injection heb ik aangepast.

Het is idd wat raar dat de __toString side effects heeft, maar die method wordt aangeroepen door twig, en dan pas kunnen we onze request processen. Initieel is de request nog niet ingevuld, waardoor we de nodige data nog niet hebben.